src/MongoDB/WriteResult.c

```diff
@@ -97,17 +102,16 @@ PHP_METHOD(WriteResult, getModifiedCount)
 		return;
 	}
 
-	if (intern->write_result.omit_nModified) {
-		RETURN_NULL();
+	if (bson_iter_init_find(&iter, intern->reply, "nModified") && BSON_ITER_HOLDS_INT32(&iter)) {
+		RETURN_LONG(bson_iter_int32(&iter));
```

Comment:
Not sure whether this is a like for like replacement. The new code doesn't use omit_nModified.